Lil-Peep/src/snippets/body-collection.liquid

```diff
@@ -42,77 +42,76 @@
+            </div>
+            <div class="product-image">
+              {% if product.featured_image != blank %}
+                <a href="{{ product.url | within: collection }}">
+                  {{ product.featured_image.src | img_url: '480x480' | img_tag: product.featured_image.alt }}
+                </a>
               {% endif %}
-
-              {% unless product.available %}
-                {{ 'products.product.sold_out' | t }}
-              {% endunless %}
-
-            </p>
+            </div>
           </div>
-          <div class="view-in-detail">
-            <a href="" type="button" name="view-button">
-              <h4>{{ 'collections.general.view' | t }}</h4>
-            </a>
+          <div class="product-data">
+            <p>
+              <a href="{{ product.url | within: collection }}" class="product-title">{{ product.title }}</a>
+            </p>
+            <div class="price">
+              <p class="price">
+                {{ product.price | money }} USD
+                {% if product.compare_at_price > product.price %}
+                  <span class="visually-hidden">{{ 'products.product.regular_price' | t }}</span>
+                  <p class="offer">{{ product.compare_at_price | money }} USD </p>
+                {% endif %}
+
+                {% unless product.available %}
+                  {{ 'products.product.sold_out' | t }}
+                {% endunless %}
+
+              </p>
+            </div>
+            <div class="view-in-detail">
+              <a href="" type="button" name="view-button">
+                <h4>{{ 'collections.general.view' | t }}</h4>
+              </a>
+            </div>
           </div>
         </div>
-      </div>
 
-    {% else %}
-      {% if collection.handle == 'all' and collection.all_vendors.size == 0 and collection.all_types.size == 0 %}
-        {% for i in (1..8) %}
-          <a href="#">
-            {% capture current %}{% cycle 1, 2, 3, 4, 5, 6 %}{% endcapture %}
-            {{ 'product-' | append: current | placeholder_svg_tag: 'placeholder-svg placeholder-svg--small' }}
-          </a>
+      {% else %}
+        {% if collection.handle == 'all' and collection.all_vendors.size == 0 and collection.all_types.size == 0 %}
+          {% for i in (1..8) %}
+            <a href="#">
+              {% capture current %}{% cycle 1, 2, 3, 4, 5, 6 %}{% endcapture %}
+              {{ 'product-' | append: current | placeholder_svg_tag: 'placeholder-svg placeholder-svg--small' }}
+            </a>
 
-          <p>
-            <a href="#">{{ 'homepage.onboarding.product_title' | t }}</a>
-          </p>
+            <p>
+              <a href="#">{{ 'homepage.onboarding.product_title' | t }}</a>
+            </p>
 
-          <p>
-            {{ 1999 | money }}
-          </p>
-        {% endfor %}
-      {% else %}
-        <p>{{ 'collections.general.no_matches' | t }}</p>
-      {% endif %}
-    {% endfor %}
+            <p>
+              {{ 1999 | money }}
```

Comment:
This has to be hardcoded? Is not available in any liquid object, maybe `product.price` or something like that?